specification/vmware/Microsoft.AVS/examples/2023-09-01/WorkloadNetworks_Get.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "WorkloadNetworks_Get",
+  "operationId": "WorkloadNetworks_Get",
+  "parameters": {
+    "api-version": "2023-09-01",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "group1",
+    "privateCloudName": "cloud1",
+    "workloadNetworkName": "default"
```

Comment:
>workloadNetworkName": "default" [](http://example.com/codeflow?start=5&length=31)

just wondering, is it meant to still be needed? Or is it a singleton?

Reply:
It is a singleton. I'll remove those entries.

Reply:
removed in 2213585eed54328a19f28437c756ba3717600095